fix: replace bitnami repo with bitnamilegacy

k9ert opened the pull request:
We need to switch out the bitnami charts but in the meantime, let's temporally use bitnamylegacy images.
https://github.com/bitnami/containers/issues/83267

Also:
* Adding [k9s](https://k9scli.io/) to the nix available binaries. Quite usefull for dev
* For me, Tilt went in a endless loop because after downloading the chart dependencies. The change in the Tiltfile prevents that by ignoring the directories where helm is downloading dependent charts.

k9ert commented:
tested in k3d. works!

dolcalmi commented on dev/Tiltfile:
why is this related?

dolcalmi commented on flake.nix:
```diff
@@ -33,6 +33,7 @@
               yq-go
               kubernetes-helm
               opentofu
+              k9s
```
same here

k9ert replied:
This is a tiny improvement doing as a sidetask. I guess i should have mentioned it in the description?

dolcalmi commented on charts/galoy/values.yaml:
are we sure these are the only required changes? what about lnd? should this not update lock file?

k9ert replied:
No, i'm not sure. But i think we should merge this ASAP. See the WIP issue for further details.

openoms commented on dev/Tiltfile:
```diff
@@ -1,8 +1,10 @@
-include('./monitoring/Tiltfile')
+#include('./monitoring/Tiltfile')
 include('./galoy/Tiltfile')
 include('./addons/Tiltfile')
-include('./stablesats/Tiltfile')
-include('./kafka-connect/Tiltfile')
+#include('./stablesats/Tiltfile')
+#include('./kafka-connect/Tiltfile')
```
uncomment these leftovers

k9ert replied:
Done

k9ert commented on charts/galoy/values.yaml:
```diff
@@ -2,6 +2,13 @@
+# ToDo: switch to non-bitnami-charts
+global:
+  security:
+    allowInsecureImages: true
```
Done

k9ert commented:
Addressed all comments but i can't test the changes right now due to:

```
 → Error: could not download oci://registry-1.docker.io/bitnamicharts/postgresql: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/bitnamicharts/postgresql/manifests/sha256:03c06d49b0f84507c9299ad7d4eaf33ffa81c02d92d4a550bc2045d5331bca88: 429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit as 'k9ert': dckr_jti_tt7e7h1Dr5SqwqfjNjySOR4ZLuo=. You may increase the limit by upgrading. https://www.docker.com/increase-rate-limit
 ```

openoms commented:
the bitnamilegacy mongodb image is not available for Apple Silicon
<img width="1098" height="128" alt="image" src="https://github.com/user-attachments/assets/a9fd0cbd-85e6-43c4-acf0-fff68a8cb014" />

k9ert commented:
> the bitnamilegacy mongodb image is not available for Apple Silicon
That's probably also a selling argument: If you want to develop on your mac, use the latest stuff.
If you want the full package, buy our secure images.

> Seems that the global flag is needed:
Ok, will revert.
